docs/WindowsBuild.md

```diff
@@ -41,17 +43,18 @@ Windows.
 
 ### 2. Clone the repositories
 1. Create a folder to contain all the Swift repositories
-1. Clone `apple/swift-cmark` into a folder named `cmark`
-1. Clone `apple/swift-clang` into a folder named `clang`
-1. Clone `apple/swift-llvm` into a folder named `llvm`
+1. Clone `apple/swift-cmark` into a folder named `swift-cmark`
+1. Clone `apple/swift-clang` into a folder named `swift-clang`
+1. Clone `apple/swift-llvm` into a folder named `swift-llvm`
+1. Clone `apple/swift-compiler-rt` into a folder named `swift-compiler-rt`
 1. Clone `apple/swift` into a folder named `swift`
 - Currently, other repositories in the Swift project have not been tested and
   may not be supported.
 
 ### 3. Build ICU
 1. Download and extract the [ICU source
 code](http://site.icu-project.org/download) to a folder named `icu` in the same
-directory as the other Swift project repositories.
+directory as the other Swift project repositories (tested with ICU version 55.1).
```

Comment:
I've tested it with 59.1.

Reply:
I'll update the documentation accordingly.